frontend/src/pages/forgotPassword/formValidation.tsx

```diff
@@ -0,0 +1,53 @@
+import { useContext, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import Form from '../../components/form';
+import { InputFieldWrapper } from '../../components/form/form-styles';
+import { InputField } from '../../components/input/InputField';
+import { ResetPasswordData } from '../../interfaces/types';
+import { AuthContext } from '../../providers/authProvider';
+import { Error } from '../../components/text/text-styles';
+import { Routes } from '../../enums/routes';
+import { emailFormValidation } from '../../validation/config/EmailFormValidation';
+
+const EmailFormValidation = () => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<ResetPasswordData>();
+
+  const validation = emailFormValidation;
+
+  const { resetPassword, isLoading } = useContext(AuthContext);
+  const [error, setError] = useState<string>('');
+
+  const submitData = (data: ResetPasswordData) => {
+    resetPassword(
+      data,
+      (navigation) => {
+        navigation(Routes.RESET_PASSWORD);
```

Comment:
I don't think that user should be sent to "reset password" page after submit. He should be sent there from link in email which he gets.

Reply:
Yeah, it's changed to display a message